lib/placebo/meck.ex

```diff
@@ -98,7 +98,7 @@ defmodule Placebo.Meck do
       |> Process.info(:dictionary)
       |> get_in([Access.elem(1), :"$callers", default_value([])])
 
-    [pid | callers]
+    List.flatten([pid] ++ [callers])
```

Comment:
```suggestion
    [pid | callers]
```

Reply:
By wrapping data in `default_value/1`, I believe this fixes the improper list you're seeing with the `[pid | callers]` syntax, so it's safe to keep it.